src/style-guide/README.md

```diff
@@ -243,40 +233,18 @@ computed: {
 </ul>
 ```
 
-We get the following benefits:
+Alternatively, we can use a `<template>` tag with `v-for` to wrap `li` element:
```

Comment:
I think this needs an article:

```text
to wrap the `li` element:
        ^^^
```

Perhaps also angle brackets around the `<li>`?

Reply:
Fixed 👍🏻